ui/apps/platform/src/hooks/useAnalytics.ts

```diff
@@ -69,7 +69,7 @@ export const COMPLIANCE_REPORT_DOWNLOAD_GENERATION_TRIGGERED =
 export const COMPLIANCE_REPORT_MANUAL_SEND_TRIGGERED = 'Compliance Report Manual Send Triggered';
 export const COMPLIANCE_REPORT_JOBS_TABLE_VIEWED = 'Compliance Report Jobs Table Viewed';
 export const COMPLIANCE_REPORT_JOBS_VIEW_TOGGLED = 'Compliance Report Jobs View Toggled';
-export const COMPLIANCE_REPORT_RUN_STATE_FILTERED = 'Compliance Report Run State Filtered';
+export const COMPLIANCE_REPORT_JOB_STATUS_FILTERED = 'Compliance Report Job Status Filtered';
```

Comment:
I don't believe these are being utilized yet, so it should be ok to change without issues. (cc: @bradr5 )